src/main/java/PostgresDriverWrapper.java

```diff
@@ -0,0 +1,66 @@
+import static java.util.Arrays.asList;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Properties;
+
+import org.apache.arrow.vector.VectorSchemaRoot;
+import org.apache.arrow.vector.types.Types;
+import org.apache.arrow.vector.types.pojo.Field;
+import org.apache.arrow.vector.types.pojo.Schema;
+import org.postgresql.Driver;
+
+public class PostgresDriverWrapper implements DBDriverWrapper{
```

Comment:
Please ALWAYS reformat your code before asking for review.

Reply:
done